src/modules/index.js

```diff
@@ -0,0 +1,14 @@
+const todoList = [
+  {
+    index: 1,
+    description: 'Wash the dishes',
+    completed: false,
+  },
+  {
+    index: 2,
+    description: 'Complete To Do List project',
+    completed: false,
+  },
+];
+
+export default todoList;
```

Comment:
[OPTIONAL] 👍🏽 

- I see you have 2 index.js files. One default, and another one as a module. I would like to kindly suggest that you consider renaming the `index.js` in the `modules` folder so as to avoid confusion and mix up with subsequent milestones in the project. 😄

Reply:
how it works Loom video link: https://www.loom.com/share/2b6b2705e3954294b1f17e896502f67d

TYPE in the input field and press ENTER